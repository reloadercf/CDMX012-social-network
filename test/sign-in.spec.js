/**
 * @jest-environment jsdom
 */
import { signInPage } from '../src/pages/sign-in.js';

jest.mock('../src/firebase-imports.js');

describe('Tests of signInPage', () => {
  it('render signInPage', () => {
    const renderSignIn = signInPage();
    document.body.innerHTML = '<div id="root"></div>';
    const rootDiv = document.getElementById('root');
    rootDiv.appendChild(renderSignIn);
    expect(rootDiv.innerHTML).toMatchSnapshot();
  });
});
