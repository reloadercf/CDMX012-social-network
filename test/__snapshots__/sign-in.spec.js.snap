// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests of signInPage render signInPage 1`] = `"<div class=\\"signInWelcomePage\\"><div class=\\"lp-content\\"><div class=\\"landing-page-container\\"><header class=\\"logo-header\\"><img class=\\"logo-book\\" src=\\"./assets/logo sin fondo 1.png\\" alt=\\"logo book reads\\"><img class=\\"titleBookReads\\" src=\\"./assets/nombre_sin_fondo-removebg-preview 1.png\\" alt=\\"titleBookReads\\"><h1 class=\\"sloganSN\\">Read with me</h1></header></div><div class=\\"landing-page-img\\"></div></div><div class=\\"enterAccount-container\\"><h2 class=\\"welcome-title\\">Welcome back</h2><form class=\\"enterAccount-form\\" id=\\"signInForm\\"><div class=\\"label-icon\\"><img class=\\"mail\\" src=\\"./assets/icons8-email-64 1.png\\" alt=\\"mail\\"><label for=\\"email\\" class=\\"e-mail\\">E-mail</label></div><input class=\\"e-mail\\" type=\\"email\\" placeholder=\\"e-mail@example.com\\" required=\\"\\" id=\\"txtEmail\\"><div class=\\"label-icon\\"><img class=\\"padlock\\" src=\\"./assets/icons8-contraseÃ±a-24 1.png\\" alt=\\"padlock\\"><label for=\\"password\\" class=\\"password\\" title=\\"Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters\\">Password</label></div><input class=\\"password\\" type=\\"password\\" placeholder=\\"Insert your password here\\" required=\\"\\" id=\\"txtPassword\\"><p class=\\"error-message\\"></p></form><input type=\\"button\\" class=\\"btn-signInWelcome\\" id=\\"btn-signInWelcome\\" value=\\"Sign In\\"><span class=\\"otherProviders\\"><label for=\\"\\" class=\\"otherProviders\\">or sign in with</label><img class=\\"logoGoogle\\" id=\\"btn-google2\\" src=\\"./assets/icons8-logo-de-google-48 2.png\\" alt=\\"logo Google\\"><img class=\\"logoFacebook\\" id=\\"btn-facebook2\\" src=\\"./assets/image 15.png\\" alt=\\"logo Facebook\\"><img class=\\"logoGithub\\" id=\\"btn-github2\\" src=\\"./assets/icons8-github-60.png\\" alt=\\"logo Github\\"></span></div></div>"`;
